models/fmcib/utils/FMCIBRunner.py

```diff
@@ -0,0 +1,53 @@
+"""
+---------------------------------------------------------
+Author: Suraj Pia
+Email:  bspai@bwh.harvard.edu
+---------------------------------------------------------
+"""
+
+import json
+import torch
+from fmcib.models import fmcib_model 
+import SimpleITK as sitk
+from mhubio.core import Instance, InstanceData, IO, Module
+from fmcib.preprocessing import preprocess
+
+
+class FMCIBRunner(Module):
+    @IO.Instance()
+    @IO.Input('in_data', 'nrrd:mod=ct', the='Input NRRD file')
+    @IO.Input('in_mask', 'nrrd|json', the='Tumor mask for the input NRRD file')
```

Comment:
In the code, we need to check which data type was provided. 
As mentioned, I suggest we outsource the centroid extraction into a module (I can draft one) to make it all more spread, verbose, and transparent (this is more of a nice-to-have than a technical requirement but I think we agreed on this :D).